.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # Devops & CI workflows
-.github/dependabot.yml      @ItsDrike
-.github/workflows/**        @ItsDrike
+.github/dependabot.yml      @py-mine-gh-actions
+.github/workflows/**        @py-mine-gh-actions
```

This should be enough, I assume you wanted to make a custom team called `py-mine-gh-actions`? With multiple code-owners specified, if just 1 of them approves, it's treated as meeting the code-owner approval requirement. I don't think we need a team here.

```suggestion
.github/dependabot.yml      @ItsDrike @py-mine-bot 
.github/workflows/**        @ItsDrike @py-mine-bot
```

---

Sorry, I meant to pop into Discord and pre-emptively ask some questions along with this proposed question:

Do we always want at least one human reviewer in addition to the bot?
Do you want to be required for all action reviews?
Does GitHub have any functionality yet like GitLab does where we can require X number per team/group owner?

---

> Do we always want at least one human reviewer in addition to the bot?

No, especially just for version bumps of the various actions, an approve from the bot is sufficient, no need for my (or someone else's) approve in addition IMO.

> Do you want to be required for all action reviews?

Not necessarily for the version bumps, but other than that, for any changes in workflows yes, and since there's no way to except the version bumps from my review with GitHub, I'm fine with being requested as a reviewer for those too, as long as it's in union with the bot, and an approve from the bot is sufficient.

> Does GitHub have any functionality yet like GitLab does where we can require X number per team/group owner?

Nope, at least not directly. There are some actions that can pass/fail based on various review conditions, so it would be possible to set up a required workflow with such an action to do get some more fine-grained review requirements, but I've looked several times for nice ones, and I've not been very satisfied with any of these options I found. Don't get me wrong, there are some pretty nice actions for this, it's just that I always find them lacking certain functionality that I'd also like and a lot of it is also just the fact that github interface doesn't allow for certain things.

But I don't really think that we need to go with this route for mcstatus, the change I suggested should be sufficient for those dependabot PRs to get auto-merged, so if that's the only goal of this PR, I think going with that route would be enough.